src/sentry/notifications/utils/participants.py

```diff
@@ -230,12 +232,44 @@ def determine_eligible_recipients(
         if team:
             return {team}
 
+    elif target_type == ActionTargetType.RELEASE_MEMBERS:
+        return get_release_committers(project, event)
```

Comment:
Log the following participants here:
* Suspect committers
* Code owners
* Anything else useful